tests/tools/onert_train/src/measure.h

```diff
@@ -34,7 +35,20 @@ uint64_t nowMicros()
 namespace onert_train
 {
 
+enum PhaseType
+{
+  MODEL_LOAD,
+  PREPARE,
+  EXECUTE,
+  END_OF_PHASE
+};
+
 struct Step
+{
+  uint64_t time; // us
```

Comment:
(optional) use `time_usec` instead of `time`. Also, I don't know what time it is in `time`. `start_time`?